src/lib/transform.js

```diff
@@ -150,6 +152,24 @@ const arTransformers = [
   'lastSaved',
 ];
 
+function validateCsvRows(csvRows) {
```

Comment:
Can we rename this function to help explain it is adding missing columns for rows without all columns? Might also warrant a comment explaining the problem (I know I'll forget about it in a month or two!)